README.md

```diff
@@ -26,8 +26,9 @@ If you would like to contribute another port (which would be awesome!), please o
 ## How to get started
 ### In the browser
 ```HTML
-<script type="text/javascript" src="VerbalExpressions.js"></script>
+<script src="VerbalExpressions.js"></script>
 ```
+Or use the [jsDelivr CDN](http://www.jsdelivr.com/projects/jsverbalexpressions).
```

Comment:
Or you could make an entire landing page with a [copy] button...
https://www.foundationcdn.com/

But is seems you guys know better than I what you want; perhaps make your
own PR?

On Thu, Jan 14, 2016 at 12:20 PM, Mihai Ionut Vilcu <
notifications@github.com> wrote:

> In README.md
> <https://github.com/VerbalExpressions/JSVerbalExpressions/pull/128#discussion_r49771301>
> :
>
> >  ```
> > +Or use the [jsDelivr CDN](http://www.jsdelivr.com/projects/jsverbalexpressions).
>
> please also add a code block that we can use to copy/paste the script on
> the page
> <script src="
> https://cdn.jsdelivr.net/jsverbalexpressions/0.2.0/verbalexpressions.min.js
> "></script>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/VerbalExpressions/JSVerbalExpressions/pull/128/files#r49771301>
> .
>